app/services/link_checker_api_service.rb

```diff
@@ -11,11 +11,23 @@ def self.check_links(reportable, webhook_uri)
     uris = extract_links(reportable)
     raise "Reportable has no links to check" unless uris.count > 0
 
+    Rails.logger.info "Calling create_batch for Edition #{reportable.id}"
+
     batch_report = Whitehall.link_checker_api_client.create_batch(
       uris,
       webhook_uri: webhook_uri,
       webhook_secret_token: webhook_secret_token
     )
+
+    Rails.logger.info "Batch for Edition #{reportable.id} with id #{batch_report.id}"
```

Comment:
Might be good to extract `batch_report.id` out to a variable because I think it should be `batch_report_id = batch_report.fetch('id')`.